Nos/Views/RelayView.swift

```diff
@@ -32,32 +32,59 @@ struct RelayView: View {
     
     var body: some View {
         List {
-            if let relays {
-                Section {
-                    ForEach(relays) { relay in
-                        Text(relay.address ?? Localized.error.string)
-                            .foregroundColor(.textColor)
+            Section {
+                ForEach(relays) { relay in
+                    Text(relay.address ?? Localized.error.string)
+                        .foregroundColor(.textColor)
+                }
+                .onDelete { indexes in
+                    Task {
+                        for index in indexes {
+                            let relay = relays[index]
+                            await relayService.closeConnection(to: relay)
+                            analytics.removed(relay)
+                            author.remove(relay: relay)
+                            viewContext.delete(relay)
+                        }
+                        
+                        try! viewContext.save()
```

Comment:
Shouldn't force try this database save operation and should show the error in an alert instead